apps/07jquery-tic-tac-toe/script.js

```diff
@@ -1,5 +1,141 @@
 'use strict';
 
 $(document).on('ready', function() {
-  // Put app logic in here
+
+    var $dataCellDiv = $('[data-cell]');
+    var $playerTurn = 'X';
```

Comment:
Since this guy's just a plain old `string`, I would use the $ character in the variable name. Save that for the jquery objects